keyboards/tgr/jane/v2ce/keymaps/default_tkl_f13_iso/keymap.c

```diff
@@ -0,0 +1,31 @@
+/* Copyright 2019 MechMerlin
```

Comment:
This keymap is redundant, it's identical to the one in `layouts/default`.

Reply:
Removed the `default_tkl_f13_iso` keymap.